test/functionalTest/cases/117_convop_using_standard_ops/getEntityByIdAttributeByName.test

```diff
@@ -0,0 +1,560 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
What about tests using pagination? Or maybe you consider pagination tested in the tests in a previous PR (https://github.com/telefonicaid/fiware-orion/pull/754)?